llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -298,16 +404,7 @@ def run_event(self, event: Event, state: State, logger: Logger) -> None:
                     + f'Got {type(state.model)} instead.',
                 )
             if self.remote_ud is not None:
-                try:
-                    self.remote_ud.init(state, logger)
-                except PermissionError as e:
-                    if 'Client Error' in str(
-                        e,
-                    ):  # thrown from composer.utils._wrap_mlflow_exceptions
-                        raise StoragePermissionError(
-                            'Error when write to save_folder.',
-                        ) from e
-                    raise e
+                self.remote_ud.init(state, logger)
```

Comment:
did you intend to delete this try/except? i assume not

Reply:
i need to remove this